tools/topology/topology2/cavs/platform/intel/deep-buffer.conf

```diff
@@ -0,0 +1,39 @@
+
+	Object.Pipeline {
+		deepbuffer-playback."1" {
+			index $DEEP_BUFFER_PIPELINE_ID
+
+			Object.Widget.copier.1 {
+				stream_name 'Deepbuffer Playback'
+			}
+
+			Object.Widget.gain.1 {
+				Object.Control.mixer.1 {
+					name 'Playback Volume 1'
+				}
+			}
+		}
+	}
+
+	Object.PCM {
+		pcm.$DEEP_BUFFER_PCM_ID {
+			name 'DeepBuffer'
+			id $DEEP_BUFFER_PCM_ID
+			direction playback
+			playback_compatible_d0i3 true
+
+			Object.Base.fe_dai.'DeepBuffer' {}
+
+			Object.PCM.pcm_caps.playback {
+				name 'Deepbuffer Playback'
+				formats 'S16_LE,S24_LE,S32_LE'
+			}
+		}
+	}
+
+	Object.Base {
+		route."1000" {
+			source  $DEEP_BUFFER_PIPELINE_SRC
+			sink	$DEEP_BUFFER_PIPELINE_SINK
+		}
+	}
```

Comment:
where is the token or parameter that defines the depth of the deep-buffer?

Reply:
https://github.com/thesofproject/sof/blob/main/tools/topology/topology2/cavs/common_definitions.conf#L60
DEEPBUFFER_FW_DMA_MS			100 # 100 ms copier dma size

Reply:
it used by https://github.com/thesofproject/sof/blob/main/tools/topology/topology2/include/pipelines/cavs/deepbuffer-playback.conf#L58

Reply:
it's very odd to me that the deep-buffer configuration is not part of the deep-buffer.conf....I would also argue that's questionable, in that all other definitions are fixed for a given architecture while the deep-buffer size is an application-dependent  or product-dependent definition.

Reply:
config it in make file